src/libtriton/api/api.cpp

```diff
@@ -1089,15 +1089,15 @@ namespace triton {
   }
 
 
-  std::unordered_map<triton::usize, triton::engines::solver::SolverModel> API::getModel(const triton::ast::SharedAbstractNode& node, triton::engines::solver::status_e* status, triton::uint32 timeout) const {
+  std::unordered_map<triton::usize, triton::engines::solver::SolverModel> API::getModel(const triton::ast::SharedAbstractNode& node, triton::engines::solver::status_e* status, triton::uint32 timeout, triton::uint32 *solving_time) const {
```

Comment:
Can you replace all `triton::uint32 *var_name` by `triton::uint32* var_name`? The pointer keyword (`*`) must be attached to its type in matter of following the coding style :)